install_on_linux.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+COMPOSE_SWITCH_VERSION="1.0.0"
+
+# FIXME Use final URLs
+COMPOSE_SWITCH_URL="https://github.com/docker/compose-switch/releases/download/$COMPOSE_SWITCH_VERSION/docker-compose"
+
+DOCKER_COMPOSE_V1_PATH_ORIG="$(which docker-compose)"
+DOCKER_COMPOSE_V1_PATH_NEW="$DOCKER_COMPOSE_V1_PATH_ORIG-v1"
+
+if [ ! -f "${DOCKER_COMPOSE_V1_PATH_ORIG}" ]; then
+  echo "'docker-compose' version 1 could not be found in PATH. Aborting..."
+  exit 1
+fi
+
+if [ -f "${DOCKER_COMPOSE_V1_PATH_NEW}" ]; then
+  echo "Looks like docker-compose v1->v2 switch is already installed. 'docker-compose' binary was already moved to '${DOCKER_COMPOSE_V1_PATH_NEW}'."
+  echo "If you are trying to re-enable Docker Compose V2, please run:"
+  echo "$ docker-compose enable-v2"
+  exit 0
+fi
+
+error=$(docker compose version 2>&1 >/dev/null)
+if [ $? -ne 0 ]; then
+  echo "Docker Compose v2 (docker-compose-plugin) is not installed"
```

Comment:
nit: I’m wondering if ` (docker-compose-plugin) `  might not confuse users, the plugin binary is named ` docker-compose `  isn’t it?
What about just
```suggestion
  echo "Docker Compose v2 is not installed"
```

Reply:
This is the name defined by https://github.com/docker/docker-ce-packaging/pull/553

Reply:
Oh OK, I’m not very familiar with using Docker on Linux.
Fair enough!